ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/support/TransactionCountResponder.java

```diff
@@ -40,7 +40,7 @@
   }
 
   private static final String ETH_REQUEST_REGEX_PATTERN = ".*eth_getTransactionCount.*";
-  private static final String EEA_REQUEST_REGEX_PATTERN = ".*eea_getTransactionCount.*";
+  private static final String EEA_REQUEST_REGEX_PATTERN = ".*priv_getEeaTransactionCount.*";
```

Comment:
We also need to test the priv_getTransactionCount for the transactions that use a privacy group id